kythe/go/indexer/cmd/go_indexer/go_indexer.go

```diff
@@ -33,27 +33,30 @@ import (
 	"kythe.io/kythe/go/indexer"
 	"kythe.io/kythe/go/platform/delimited"
 	"kythe.io/kythe/go/platform/kzip"
+	"kythe.io/kythe/go/util/flagutil"
 	"kythe.io/kythe/go/util/metadata"
 
 	apb "kythe.io/kythe/proto/analysis_go_proto"
 	spb "kythe.io/kythe/proto/storage_go_proto"
 )
 
 var (
-	doJSON         = flag.Bool("json", false, "Write output as JSON")
-	doLibNodes     = flag.Bool("libnodes", false, "Emit nodes for standard library packages")
-	doCodeFacts    = flag.Bool("code", false, "Emit code facts containing MarkedSource markup")
-	doAnchorScopes = flag.Bool("anchor_scopes", false, "Emit childof edges to an anchor's semantic scope")
-	metaSuffix     = flag.String("meta", "", "If set, treat files with this suffix as JSON linkage metadata")
-	docBase        = flag.String("docbase", "http://godoc.org", "If set, use as the base URL for godoc links")
-	verbose        = flag.Bool("verbose", false, "Emit verbose log information")
-	contOnErr      = flag.Bool("continue", false, "Log errors encountered during analysis but do not exit unsuccessfully")
+	deriveFileMetaEdgeKinds flagutil.StringList
```

Comment:
When would this be anything other than the empty list or `["/kythe/edge/generates"]`?

Reply:
Don't have a specific case; just felt that a boolean was too restrictive, but happy to clamp it down to a boolean.

Reply:
I don't think even the `bool` is needed.  Why gate the behavior?

Reply:
For C++ and Java, I was able to restrict the derivation of file-file generates edges to just the proto metadata. For Go, that wasn't possible (there's no proto metadata implementation externally).

I'm not attached to the gating, though.

Reply:
I don't think this should be a gated feature.

Reply:
Removed